### PR TITLE
Feature upload files

### PR DESCRIPTION
**Pull request by Lola-D**
On this branch:

- mutation to create a task, a render and a correction
- we need to create a connexion to Cloudinary because files will be hosted on it, then we receive an secure url from Cloudinary to put into our new task document
- for the .sample-test-task file, since I can't find to way to mock Cloudinary connexion and request, I pushed it anyway in this request to save the correct part in case we will find the good way to make it work

### CODE REVIEW

File: src/types/allTypes.ts [hunk 1]
```diff
@@ -44,7 +45,8 @@ export type RenderType = {
   _id: mongodb.ObjectID;
   user: mongodb.ObjectID;
   task: mongodb.ObjectID;
-  creation_date: Date;
+  url: string;
```
Comment (Lola-D) [inline src/types/allTypes.ts:48]: it is on the Render type and not on AssignationTask type

File: src/graphql/resolvers.ts [hunk 2]
```diff
@@ -1,127 +1,94 @@
...
-//     const currentUser: any = await User.findById(parent.id);
-//     const roleUser: any = await Role.findById(currentUser.role);
-//     return roleUser;
-//   },
-// },
+  Mutation: {
+    createTask: async (parent: any, args: any): Promise<Document> => {
+      cloudinaryConfig();
+      try {
+        const result = await cloudinary.v2.uploader.upload_large(
```
Comment (Lola-D) [inline src/graphql/resolvers.ts:30]: add an new try catch to verify cloudinary request

File: src/graphql/resolvers.ts [hunk 3]
```diff
@@ -1,127 +1,94 @@
...
-// import { Role } from "../models/role";
-// import { Speciality } from "../models/speciality";
-// import { ClassroomType, TaskType, UserType } from "allTypes";
-// import { ApolloError } from "apollo-server-express";
-// import { Classroom } from "../models/classroom";
+import { Render } from "../models/render";
+import { Correction } from "../models/correction";
+import { RenderType, TaskType, CorrectionType } from "allTypes";
+
+const cloudinaryConfig = () => {
```
Comment (Lola-D) [inline src/graphql/resolvers.ts:11]: put on an utils folder

File: src/graphql/resolvers.ts [hunk 4]
```diff
@@ -1,127 +1,94 @@
...
-//       return e.message;
-//     }
-//   },
-// },
-// };
+    createCorrection: async (parent: any, args: any): Promise<Document> => {
+      cloudinaryConfig();
+      try {
+        const result = await cloudinary.v2.uploader.upload(args.input.url, {
+          allowed_formats: ["jpg", "png"],
```
Comment (Lola-D) [inline src/graphql/resolvers.ts:75]: only pdf

**Review by Jayjay59 — APPROVED** [review]

**Review by clrko — APPROVED** [review]
As discussed, great

**Review by eQuantin — APPROVED** [review]

**Pull request merged**